demo/client/main.html

```diff
@@ -1,23 +1,28 @@
 <head>
-  <title>Prosemeteor</title>
+ <title>ProseMeteor</title>
 </head>
 
 <body>
-  <h1>Prosemeteor Package</h1>
-  <p>
-    Below are two separate ProseMeteorEditor instances. Each will connect to the AuthorityManager on the server
-    and ask it to open its document, which in turn will make the AuthorityManager create an Authority to start tracking
-    that individual document.
-  </p>
+ <h1>Prosemeteor Package</h1>
+ <p>
+   Below are two separate ProseMeteorEditor instances. Each will connect to the AuthorityManager on the server
+   and ask it to open its document, which in turn will make the AuthorityManager create an Authority to start tracking
+   that individual document.
+ </p>
 
-  <h3>Document id 'proofOfConceptDocId1'</h3>
-  <div class="prosemeteor-demo" id="prosemeteor-poc-1">
-  </div>
+ <section>
+   <h3>Document id 'proofOfConceptDocId1'</h3>
+   <div class="prosemeteor-demo" id="prosemeteor-poc-1"></div>
+ </section>
 
-  <hr />
+  <section>
+    <h3>Document id 'proofOfConceptDocId2'</h3>
+    <div class="prosemeteor-demo" id="prosemeteor-poc-2"></div>
+  </section>
 
-  <h3>Document id 'proofOfConceptDocId2'</h3>
-  <div class="prosemeteor-demo" id="prosemeteor-poc-2">
-
-  </div>
+  <section>
```

Comment:
new doc in the demo app that should fail authentication and call the `onAuthenticationError()` callback